app/src/main/java/hr/foi/air/servicesync/ui/components/CompanyLocation.kt

```diff
@@ -3,16 +3,32 @@ package hr.foi.air.servicesync.ui.components
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
-import com.example.maps.interfaces.IMapProvider
 import com.google.firebase.firestore.GeoPoint
+import hr.foi.air.servicesync.R
+import hr.foi.air.servicesync.business.MapProviderManager
 
 @Composable
-fun CompanyLocation(geoPoint: GeoPoint, mapProvider: IMapProvider)
+fun CompanyLocation(geoPoint: GeoPoint)
 {
-    Box(modifier = Modifier.fillMaxWidth().height(200.dp)) {
-        mapProvider.CreateMap(geoPoint.latitude,geoPoint.longitude)
+    val mapProvider = MapProviderManager.getCurrentMapProvider(context = LocalContext.current)
```

Comment:
Nice usage of a new cool function. When there is no map providers there's no maps, makes sense, nice!